dotbot/dispatcher.py

```diff
@@ -35,6 +35,7 @@ def dispatch(self, tasks):
                     self._context.set_defaults(task[action]) # replace, not update
                     handled = True
                     # keep going, let other plugins handle this if they want
+                self._load_plugins()
```

Comment:
This is certainly not the best solution, it will load the plugins on each outer loop iteration. So I'm open to different ideas to make this less of a hammer.